ash/src/extensions/ext/shader_object.rs

```diff
@@ -8,21 +8,34 @@ use core::ptr;
 
 impl crate::ext::shader_object::Device {
     /// <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/vkCreateShadersEXT.html>
+    ///
+    /// When this function returns, whether or not it succeeds, it is guaranteed that every returned
+    /// element is either [`vk::ShaderEXT::null()`] or a valid [`vk::ShaderEXT`] handle.
```

Comment:
is it intentional that this one case has different docs?

Reply:
Yeah, I mentioned that in the updated commit-message/PR-description: upstream doesn't even reference the multi pipeline creation chapter. Probably because of the subject conflicts (pipeline != shader object even though the functions behave identically in these cases).

The upstream docs are terse enough that it makes sense to include them here, I think.

Reply:
SGTM.